lib/protocol/protocol.js

```diff
@@ -81,7 +93,13 @@ function _encodeFetchRequest (clientId, correlationId, payloads, maxWaitMs, minB
 
 function decodeFetchResponse (cb, maxTickMessages) {
   return function (resp) {
-    return _decodeFetchResponse(resp, cb, maxTickMessages);
+    return _decodeFetchResponse(resp, cb, maxTickMessages, 0);
```

Comment:
Version 0? 1 and 2 is used for encode, but 0 and 1 for decode.

function should be renamed too? (and deprecate the unversioned export symbol)

Reply:
Oh, I see there is literally 3 pairings now, so there is a literal v0, and v2 fetch uses a v1 response.